benchmarks/hamming/dune

```
@@ -0,0 +1,6 @@
+
+(executable
+  (name hamming)
+  (modes byte exe))
+  (alias (name hamming)
```

```suggestion
  (alias (name buildbench)
```
`buildbench` is the alias used for all sequential benchmarks